crates/bevy_ecs/src/observer/runner.rs

```diff
@@ -444,6 +461,7 @@ fn hook_on_add<E: Event, B: Bundle, S: ObserverSystem<E, B>>(
         }
     });
 }
+
```

Comment:
The whitespace changes here and below look like they slipped in by accident.  I'd be inclined to revert them, but they're also harmless.